.github/workflows/release-publish.yml

```diff
@@ -134,12 +134,19 @@ jobs:
           -a "${ARTIFACTS}" \
           -g ":nessie-quarkus" \
           -p "servers/quarkus-server" \
+          -d "Dockerfile-server" \
           ghcr.io/projectnessie/nessie
 
         # Add version to the openapi file name
         cp api/model/build/generated/openapi/META-INF/openapi/openapi.yaml api/model/build/nessie-openapi-${RELEASE_VERSION}.yaml
 
         cp gc/gc-tool/build/executable/nessie-gc gc/gc-tool/build/executable/nessie-gc-${RELEASE_VERSION}
+        tools/dockerbuild/build-push-images.sh \
+          -a "${ARTIFACTS}" \
+          -g ":nessie-gc-tool" \
+          -p "gc/gc-tool" \
+          -d "Dockerfile-gctool" \
+          ghcr.io/projectnessie/nessie-gc
```

Comment:
@snazy This would end up as a new repo in Quay and DockerHub as well. Do we need to create those repos upfront? Do we need to change regsync settings?

Reply:
The `regsync.yml` file needs to be changed, yes.

I _think_ we don't need to create repos upfront. Let's see whether the snapshot-publish job has something to complain - I guess not.